app.js

```diff
@@ -61,7 +59,7 @@ function secureCookies(req, res, next) {
   const cookie = res.cookie.bind(res);
   res.cookie = (name, value, options) => {
     options = options || {};
-    options.secure = (req.protocol === 'https');
+    options.secure = req.protocol === 'https';
```

Comment:
I totally agree - I've said it before and I'll say it again - https://github.com/feross/standard